ddtrace/appsec/processor.py

```diff
@@ -89,15 +89,22 @@ class _Addresses(object):
     "x-real-ip",
 }
 
-_COLLECTED_HEADER_PREFIX = "http.request.headers."
 
-
-def _set_headers(span, headers):
-    # type: (Span, Dict[str, Union[str, List[str]]]) -> None
+def _set_headers(span, kind, headers, extra_collect={}):
```

Comment:
```suggestion
def _set_headers(span, kind, headers, extra_collect=None):
```
Never use `{}` as a default value for a kwarg, it's dangerous. It's going to be the same object for every call.
It's an antipattern in Python.

Reply:
Gotcha! Thanks

Reply:
Fixed